include/zephyr/bluetooth/audio/bap.h

```diff
@@ -154,6 +154,53 @@ enum bt_bap_ascs_reason {
 	BT_BAP_ASCS_REASON_CIS = 0x0a,
 };
 
+/** @brief Structure storing values of fields of ASE Control Point notification. */
+struct bt_bap_ascs_rsp {
+	/** Value of the Response Code field. */
+	enum bt_bap_ascs_rsp_code code;
+
+	/** Value of the Reason field.
+	 *  The meaning of this value depend on the Response Code field.
+	 */
+	union {
```

Comment:
Do we verify that the `code/{reason | metadata}` pair is valid when we get it from the application, before sending them to the remote device?

Can/should we?

Reply:
We pass the response as soon as we receive it from the application layer callback. For now, I'd leave it that way until there's a use case for more strict behavior.

Reply:
That's fine with me; not sure what we can do about either, except change the return code and/or reason to `BT_BAP_ASCS_RSP_CODE_UNSPECIFIED`